crates/holochain_conductor_api/src/config/interface.rs

```diff
@@ -20,8 +20,8 @@ pub struct AdminInterfaceConfig {
 ///
 /// [`ConductorConfig`]: crate::conductor::ConductorConfig
 /// [`ConductorState`]: https://docs.rs/holochain/latest/holochain/conductor/state/struct.ConductorState.html
-#[derive(Clone, Deserialize, Serialize, Debug, PartialEq, JsonSchema)]
-#[serde(tag = "type", rename_all = "snake_case")]
+#[derive(Clone, Deserialize, Serialize, Debug, PartialEq)]
+#[serde(tag = "type", content = "value", rename_all = "snake_case")]
```

Comment:
Note: This one seems to conflict with the `JsonSchema` trait from the `schemars` crate so I'm forced to remove it again..